### ISSUE TITLE
Command to list available pairs

### ISSUE THREAD

**Issue by t0neg** [zertrin/clikraken#4]
Hi, 

It would be nice to have a command that would list all available trading pairs. Also maybe with some additional information from this method: [Get tradable asset pairs](https://www.kraken.com/help/api#get-tradable-pairs)

BR,
t0neg

**Comment by zertrin** [comment]
Indeed, I already thought about that, but didn't really find a solution concerning the fact that it is a pretty long list.

Any ideas?

**Comment by t0neg** [comment]
I think 41 pair isn't so long. At powershell it looks ok.
``` python
import requests
from tabulate import tabulate

r = requests.get('https://api.kraken.com/0/public/AssetPairs')
data = r.json()['result']
output = []
for c in data:
    if c.endswith('.d'):
        continue
    output.append([c,
                   data[c]['altname'],
                   data[c]['base'],
                   data[c]['quote']])

print(tabulate(output, headers=['Pair', 'Alt Name', 'Base', 'Quote']))
print('--- Total: {} pairs'.format(len(output)))
```
Output:
```
Pair      Alt Name    Base    Quote
--------  ----------  ------  -------
DASHEUR   DASHEUR     DASH    ZEUR
DASHUSD   DASHUSD     DASH    ZUSD
DASHXBT   DASHXBT     DASH    XXBT
USDTZUSD  USDTUSD     USDT    ZUSD
XETCXETH  ETCETH      XETC    XETH
XETCXXBT  ETCXBT      XETC    XXBT
XETCZEUR  ETCEUR      XETC    ZEUR
XETCZUSD  ETCUSD      XETC    ZUSD
XETHXXBT  ETHXBT      XETH    XXBT
XETHZCAD  ETHCAD      XETH    ZCAD
XETHZEUR  ETHEUR      XETH    ZEUR
XETHZGBP  ETHGBP      XETH    ZGBP
XETHZJPY  ETHJPY      XETH    ZJPY
XETHZUSD  ETHUSD      XETH    ZUSD
XICNXETH  ICNETH      XICN    XETH
XICNXXBT  ICNXBT      XICN    XXBT
XLTCXXBT  LTCXBT      XLTC    XXBT
XLTCZEUR  LTCEUR      XLTC    ZEUR
XLTCZUSD  LTCUSD      XLTC    ZUSD
XMLNXETH  MLNETH      XMLN    XETH
XMLNXXBT  MLNXBT      XMLN    XXBT
XREPXETH  REPETH      XREP    XETH
XREPXXBT  REPXBT      XREP    XXBT
XREPZEUR  REPEUR      XREP    ZEUR
XREPZUSD  REPUSD      XREP    ZUSD
XXBTZCAD  XBTCAD      XXBT    ZCAD
XXBTZEUR  XBTEUR      XXBT    ZEUR
XXBTZGBP  XBTGBP      XXBT    ZGBP
XXBTZJPY  XBTJPY      XXBT    ZJPY
XXBTZUSD  XBTUSD      XXBT    ZUSD
XXDGXXBT  XDGXBT      XXDG    XXBT
XXLMXXBT  XLMXBT      XXLM    XXBT
XXLMZEUR  XLMEUR      XXLM    ZEUR
XXLMZUSD  XLMUSD      XXLM    ZUSD
XXMRXXBT  XMRXBT      XXMR    XXBT
XXMRZEUR  XMREUR      XXMR    ZEUR
XXMRZUSD  XMRUSD      XXMR    ZUSD
XXRPXXBT  XRPXBT      XXRP    XXBT
XZECXXBT  ZECXBT      XZEC    XXBT
XZECZEUR  ZECEUR      XZEC    ZEUR
XZECZUSD  ZECUSD      XZEC    ZUSD
--- Total: 41 pairs
```

**Comment by zertrin** [comment]
Done (mostly by using your snippet of code) in commit 7d1b029c released in version 0.6.0

Thanks!
